src/btreemap.rs

```diff
@@ -928,15 +943,22 @@ mod test {
         (vec![x], vec![])
     }
 
+    // Make `Vec<u8>` bounded so that it can be used as a key/value in the btree.
+    impl BoundedStorable for Vec<u8> {
+        fn max_size() -> u32 {
+            10
+        }
+    }
+
     #[test]
     fn init_preserves_data() {
```

Comment:
Maybe I missed it, but shouldn't there be a (new) test somewhere that checks that the `BTreeMap` will complain if you try to load from memory but the bounds of the types provided are too large?

Reply:
I was admittedly being lazy. Thanks for calling me out.